### ISSUE TITLE
Markbot Issue

### ISSUE THREAD

**Issue by Hath0013** [Hath0013/fastness-fixer#1]
@thomasjbradley

Sorry I always for get to make an issue rather than email you. I'm using 5.0.2.

**Comment by thomasjbradley** [comment]
It's stuck in "computing" mode after all the tests on the left have finished?
And every time too? Now matter whether you refresh or not?

**Comment by Hath0013** [comment]
Yep, exactly that. It was always stuck, even if the tests weren’t finished. I had to quit it and open it every time I wanted to check my code.

**Comment by thomasjbradley** [comment]
Hmm... try this version out and see if it helps:
https://dl.dropboxusercontent.com/u/2506703/Markbot/Markbot-5.0.3-prerelease.dmg

**Comment by Hath0013** [comment]
hmmm got this message..

<img width="1280" alt="screen shot 2016-11-22 at 2 30 38 pm" src="https://cloud.githubusercontent.com/assets/16677645/20538706/4f9b1d80-b0c0-11e6-8b4d-eb500ed2e780.png">

**Comment by Hath0013** [comment]
opened the link through my email and it worked. one sec, I'll see if Markbot approves

**Comment by thomasjbradley** [comment]
Okay—I think it may have not quite uploaded when you tried.

**Comment by Hath0013** [comment]
Okay that seems to have worked, however I'm getting an error now I wasn't getting before. Not too sure how to go about fixing it, I'm not actually able find where to simulate slower internet speeds in Chrome.

<img width="933" alt="screen shot 2016-11-22 at 2 44 43 pm" src="https://cloud.githubusercontent.com/assets/16677645/20539222/400d4a58-b0c2-11e6-9de5-123526879fa7.png">

<img width="1280" alt="screen shot 2016-11-22 at 3 01 36 pm" src="https://cloud.githubusercontent.com/assets/16677645/20539853/9bb7d95c-b0c4-11e6-9964-26b9f31aac33.png">

**Comment by thomasjbradley** [comment]
To simulate in Chrome go to "Network" , then in the next row go to "No throttling".
The only difference is that Chrome's "Wi-Fi" network is much faster than the one built into Markbot—it’s actually faster than my home internet!

Using the `4G` network in Chrome will give you closer, but more similar results. Probably still a little slower than Markbot though.

I've also found that if the performance test is really slow the first time—refresh Markbot again (without quitting it) and it'll get faster. The may Markbot works, the first time it runs a test it has to load in a lot of new code—and that seems to affect the performance test for some reason. (I haven't figured out why yet.)

**Comment by Hath0013** [comment]
I'm not sure how to fix the error other than deleting code so there isn't so much to load. But even doing that is making the load time go up. The fonts are taking the most time to load. Can't seem to get the load time to go below 1631ms now.

**Comment by thomasjbradley** [comment]
Hmm... I wonder what's causing the delay. Compared to my solution yours even has a smaller "Size"—mine's at 379 kB.

Is this a school or at home? I wonder if the internet connection itself is causing problems.

**Comment by Hath0013** [comment]
I'm at home and on WiFi (25MBPS). Can't actually plug in with an ethernet cable because I don't have an adapter unfortunately. I only got that error after running the code through the new version you sent for some reason. It keeps fluctuating between 1600 and over 2000 when I refresh Markbot.

**Comment by thomasjbradley** [comment]
In Chrome if you emulate a 4G network what's the download time you're getting?
And what about a simulated WiFi network in Chrome?
